internal/pkg/cli/cmd/cmd.go

```diff
@@ -291,51 +291,35 @@ func (root *RootCommand) printError(errRaw error) {
 
 	// Iterate over errors and replace message if needed
 	modifiedErrs := utils.NewMultiError()
+	var errDirNotFound dependencies.DirNotFoundError
 	for _, err := range originalErrs.Errors {
 		switch {
-		case errors.Is(err, dependencies.ErrProjectDirFound):
-			root.logger.Infof(`The path "%s" is an project directory.`, root.fs.BasePath())
-			root.logger.Info(`Please use an empty directory.`)
-			if root.CalledAs() == `init` {
+		case errors.As(err, &errDirNotFound):
+			root.logger.Infof(`The path "%s" is %s.`, root.fs.BasePath(), errDirNotFound.Found())
+			if root.CalledAs() == `init` && errDirNotFound.Found() == dependencies.KbcProjectDir {
+				root.logger.Infof(`Please use %s.`, errDirNotFound.Expected())
 				root.logger.Info(`Or synchronize the current directory with the "pull" command.`)
+			} else if errDirNotFound.Expected() == dependencies.KbcProjectDir {
+				root.logger.Infof(`Please change working directory to %s.`, errDirNotFound.Expected())
+				root.logger.Infof(`Or use the "sync init" command in %s.`, dependencies.EmptyDir)
+			} else {
+				root.logger.Infof(`Please use %s.`, errDirNotFound.Expected())
+			}
+			if errDirNotFound.Expected() == dependencies.EmptyDir {
+				modifiedErrs.Append(fmt.Errorf("directory is not empty"))
+			} else {
+				modifiedErrs.Append(fmt.Errorf("neither this nor any parent directory is %s", errDirNotFound.Expected()))
```

Comment:
Tu je spracovavanie chyb, specialne pre CLI.
Tj. vypisujeme tam nejake podrobnosti a co sa ma stat, aby to islo.

Vytvoril som novy type `ErrProjectDirFound`, ktory obsahuje `expected` a `found`, tj. ak typ adresara sa ocakava a aky sa nasiel (vid dalej).

Predtym bola kazda chyba riesena samostatne, ... ale ako pribudol `dbt project dir`, tak by to bolo uz vela moznosti (kombinatorika), preto sa to riesi takto genericky, ... vid E2E testy dalej.